lib/fifo.js

```diff
@@ -145,9 +145,15 @@ class fifo {
       this._pick.bind( this ) )
 
     this._callcount++
+
+    if ( options.positionstalecall ) {
```

Comment:
I don't understand what this is for?

Reply:
Because we decided that it should hangup if the user didn't choose to cancel the callback unless I'm missing something

Reply:
Would it be a cleaner interface to simply pick the call from babble-sip, i.e.

const c = queues.findbycallerid( { callerid: "0123" } ) 
if( c ) c.pick()

Reply:
are you implying implementing findbycallerid then?

Reply:
okay...gotcha I updated the code